app/scripts/services/api-explorer.js

```diff
@@ -527,6 +525,23 @@
                     return deferred.promise;
                 },
                 /**
+                 * Get the local API group overview
+                 */
+                getLocalAPIGroupOverview : function(){
```

Comment:
I had to study the code to understand that what this method was in fact doing was returning the local URL of the group overview if it exists.  Please change the comment and/or function name so that it clearly says that it is getting the local overview path as opposed to the body of the overview itself.